src/Sprites/IndexedBmpWriter.cpp

```diff
@@ -38,10 +38,9 @@ void IndexedBmpWriter::WriteHeaders(Stream::SeekableWriter& seekableWriter, uint
 
 unsigned int IndexedBmpWriter::CalculatePitch(uint16_t bitCount, int32_t width)
 {
-	const uint16_t bitsPerByte = 8;
+	const std::size_t bitsPerByte = 8;
 	// Does not include padding
-	const uint16_t bytesOfPixelsPerRow = ((width * bitCount) + (bitsPerByte - 1)) / bitsPerByte;
-
+	const std::size_t bytesOfPixelsPerRow = ((width * bitCount) + (bitsPerByte - 1)) / bitsPerByte;
```

Comment:
Yes, actually, using std::size_t makes a lot of sense here. The pitch is used in array access, and is naturally limited by the size of memory. Perhaps we should update the return type of the function to match.